### PR TITLE
Fix URL and handling of data returned

### PR DESCRIPTION
**Pull request by jburel**
Fix URL and handling of data returned

This is required by QueryIDRWithGeneLists.ipynb
We will need a new release of idr-py
cc @manics @joshmoore @sbesson

### CODE REVIEW

**Comment by sbesson** [comment]
Assuming this has been successfully tested (and validated by Travis), no objections to seeing this released as `0.2.1`.

File: idr/externalDBs.py [hunk 1]
```diff
@@ -85,26 +85,15 @@ def get_ensembleid(gene):
...
 
-    url = ('https://www.ebi.ac.uk/QuickGO-Old/GAnnotation?tax=' +
+    url = ('https://www.ebi.ac.uk/QuickGO/services/annotation/downloadSearch?taxonId=' +
            taxonomy_id +
-           '&relType=IP&goid=%20' +
+           '&goId=%20' +
            go_term +
            '%20&format=tsv')
-    res = requests.get(url)
+    res = requests.get(url, headers={ "Accept" : "text/tsv")
```
Comment (manics) [inline idr/externalDBs.py]: You're missing a `}`

**Comment by jburel** [comment]
@manics fixed

**Comment by manics** [comment]
Functionally this seems fine.

When running the `Query IDR for Phenotypes` section of QueryIDRWithGeneLists.ipynb I noticed some white space being printed before the table.

Also some formatting issues:
```
idr/externalDBs.py:88:80: E501 line too long (88 > 79 characters)
idr/externalDBs.py:93:38: E201 whitespace after '{'
idr/externalDBs.py:93:47: E203 whitespace before ':'
```

**Pull request merged**